canonical-web-and-design/maas-ui
# Add RSD form to Add hardware dropdown.

## Done
- Added RSD form to Add hardware dropdown
- Added actions/reducers/selectors for pods

## QA
- Go to the machine list, click Add hardware then RSD and check that the form shows up
- Fill in the form, submit, and check that you get an error (unless you've done the setup required to get an RSD working... which you probably haven't).
- Check the network request is the same as if you'd done it at /#/rsd, *except* for `default_pool` which as far as I can tell from the api doesn't actually do anything because if a pool isn't sent the api assumes 0 anyway.

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1901

## Screenshot
![Screenshot_2020-04-29 Add pod bolla MAAS](https://user-images.githubusercontent.com/25733845/80560425-3f14b980-8a24-11ea-9b0d-ca79450a2b0e.png)
